From 0e66dd154b77a52465a671d4fca35d7f92de630d Mon Sep 17 00:00:00 2001
From: Isaiah Norton <isaiah@tiledb.io>
Date: Wed, 3 Feb 2021 15:06:52 -0500
Subject: [PATCH] Loosen NumPy pin for conda (1.16.6 is latest)

---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 99eebe1..24b3421 100644
--- a/setup.py
+++ b/setup.py
@@ -483,7 +483,7 @@ def cmake_available():
 
 def setup_requires():
     req = ["cython>=0.27",
-           "numpy==1.16.5 ; python_version < '3.9'",
+           "numpy==1.16.* ; python_version < '3.9'",
            "numpy ; python_version >= '3.9'",
            "setuptools>=18.0",
            "setuptools_scm>=1.5.4",
@@ -695,7 +695,7 @@ setup(
     ext_modules=__extensions,
     setup_requires=setup_requires(),
     install_requires=[
-        'numpy>=1.16.5',
+        'numpy>=1.16',
         'wheel>=0.30'
     ],
     tests_require=TESTS_REQUIRE,
-- 
2.29.2

