From 01aae208e1fcbb503e41f533b785126107e95c6f Mon Sep 17 00:00:00 2001
From: Isaiah Norton <isaiah@tiledb.io>
Date: Fri, 19 Jul 2019 09:27:49 -0400
Subject: [PATCH] Patch 0.4.3 runtime_library_dirs on windows

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index cb71ee8..10153d5 100644
--- a/setup.py
+++ b/setup.py
@@ -467,7 +467,6 @@ cy_extension=Extension(
     define_macros=DEF_MACROS,
     sources=SOURCES,
     library_dirs=LIB_DIRS,
-    runtime_library_dirs=LIB_DIRS,
     libraries=LIBS,
     extra_link_args=LFLAGS,
     extra_compile_args=CXXFLAGS,
-- 
2.20.1

