diff --git a/tiledb/tests/cc/test_cc.py b/tiledb/tests/cc/test_cc.py
index 9c5c58b..d6d6ebf 100644
--- a/tiledb/tests/cc/test_cc.py
+++ b/tiledb/tests/cc/test_cc.py
@@ -81,83 +81,6 @@ def test_enums():
     check_enum("QueryStatus")
 
 
-def test_array():
-    uri = tempfile.mkdtemp()
-
-    # TODO BOOTSTRAP
-    tiledb.from_numpy(uri, np.random.rand(4)).close()
-    # lt.from_numpy(uri, np.random.rand(4)).close()
-
-    ctx = lt.Context()
-    arr = lt.Array(ctx, uri, lt.QueryType.READ)
-    assert arr._is_open()
-    assert os.path.basename(arr._uri()) == os.path.basename(uri)
-    assert arr._schema == arr._schema
-
-    arr._reopen()
-    arr._set_open_timestamp_start(0)
-    arr._set_open_timestamp_end(1)
-    arr._reopen()
-    assert arr._open_timestamp_start == 0
-    assert arr._open_timestamp_end == 1
-
-    arr._close()
-    assert not arr._is_open()
-
-    arr = lt.Array(ctx, uri, lt.QueryType.READ)
-
-    # TODO test
-    # consolidate sig1
-    # consolidate sig2
-    # vacuum
-    # load_schema
-    # create
-    lt.Array._encryption_type(ctx, uri) == lt.EncryptionType.NO_ENCRYPTION
-    # TODO assert lt.Array.load_schema(ctx, uri) == arr._schema
-    assert arr._query_type() == lt.QueryType.READ
-
-    arr._close()
-    ####
-    arr = lt.Array(ctx, uri, lt.QueryType.WRITE)
-    arr._set_open_timestamp_start(1)
-    arr._set_open_timestamp_end(1)
-    arr._close()
-
-    arr._open(lt.QueryType.WRITE)
-    data = b"abcdef"
-    arr._put_metadata("key", lt.DataType.STRING_ASCII, len(data), data)
-    arr._close()
-
-    arr._set_open_timestamp_start(1)
-    arr._set_open_timestamp_end(1)
-    arr._open(lt.QueryType.READ)
-    assert arr._metadata_num() == 1
-    assert arr._has_metadata("key")
-    mv = arr._get_metadata("key")
-    assert mv == (data, lt.DataType.STRING_ASCII)
-
-    assert arr._get_metadata_from_index(0)[0] == lt.DataType.STRING_ASCII
-    mv = arr._get_metadata_from_index(0)[1]
-    assert bytes(mv) == data
-    with pytest.raises(lt.TileDBError):
-        arr._get_metadata_from_index(1)
-    arr._close()
-
-    arr._open(lt.QueryType.WRITE)
-    arr._set_open_timestamp_start(2)
-    arr._set_open_timestamp_end(2)
-    arr._delete_metadata("key")
-    arr._close()
-
-    arr._set_open_timestamp_start(3)
-    arr._set_open_timestamp_end(3)
-    arr._open(lt.QueryType.READ)
-    with pytest.raises(KeyError):
-        arr._get_metadata("key")
-    assert not arr._has_metadata("key")
-    arr._close()
-
-
 def test_consolidate_fragments():
     uri = tempfile.mkdtemp()
     ctx = lt.Context()
